src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import {
+  Footer,
+  ButtonContainer,
+  FooterText,
+  AboutText,
+  ModalButton,
+} from './ChartFooter.style';
+import NewDialog from 'components/NewDialog/NewDialog';
+import { Link } from 'cms-content/modals';
+import { MobileOnly, DesktopOnly } from '../Shared/Shared.style';
+import ShareButtons from 'components/LocationPage/ShareButtons';
+import { Region } from 'common/regions';
+import type { MetricValues } from 'common/models/Projections';
+
+export interface AboutModalProps {
+  header: string;
+  body: string;
+  links: Link[];
+}
+
+export interface ShareButtonProps {
+  region: Region;
+  stats: MetricValues;
+  chartIdentifier: number;
+  showEmbedButton?: boolean;
+}
+
+const MetricChartFooter: React.FC<{
+  footerText: string | null;
+  disclaimer: string | null;
+  shareButtonProps: ShareButtonProps;
+  aboutModal: AboutModalProps;
+}> = ({ footerText, disclaimer, shareButtonProps, aboutModal }) => {
+  const [openModal, setOpenModal] = useState(false);
+  const dialogProps = {
+    open: openModal,
+    closeDialog: () => setOpenModal(false),
+    header: aboutModal.header,
+    body: aboutModal.body,
+    links: aboutModal.links,
+  };
+  return (
+    <>
+      <DesktopOnly>
+        <Footer>
+          <FooterText>
+            {footerText}&nbsp;
+            <ModalButton onClick={() => setOpenModal(true)}>
+              <AboutText>About this data</AboutText>
+            </ModalButton>
+            {disclaimer && (
+              <>
+                <br />
+                <br />
```

Comment:
i think leaving the line break to css rather than two `<br/>` might render a little cleaner//allow us to control the amount of spacing (the space created by two `<br />` feels a little too large). adding a wrapping element with margin around the `disclaimer` would work